jenkins-integration/beaker/install/shared/23_install_system_gems.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+test_name 'Install System Gems'
+
+def install_system_gem(host, gem_name)
+  step "Install '#{gem_name}' gem"
+  # Ensure it's the system ruby
+  on(host, "/usr/bin/env gem install #{gem_name}")
+end
+
+step 'Install gems using system ruby'
+
+gem_list = ENV['PUPPET_GATLING_SYSTEM_GEMS'].split(',')
```

Comment:
@jpinsonault, sorry for not bringing this up earlier.  One of the things that I hadn't thought through with this approach of installing system gems is that we could arguably be polluting the state of the SUT by installing gems onto it which are just for infrastructure / setup purposes - and not part of the actual simulation that we are actually running.  To that end, an alternative approach we might want to consider for this is:

1) Just add the hocon gem dependency to the appropriate `jenkins-integration` Gemfile(s) where we're already installing `beaker` as part of the Jenkins slave setup for the test run.  This should make the hocon CLI tool available on the Jenkins slave.

2) In the `set_hocon_setting` / `unset_hocon_setting` functions, scp the file from the master down to the Jenkins slave, run the hocon CLI tool to make modifications, and scp the modified file back to the master.

This is a bit more inefficient than modifying the file in place on the master but I don't think the overhead would be too big of a deal.

I talked through this alternative approach with @cprice404 and I think he's supportive of it, for the benefit of reducing the amount of pollution of SUT state.  What do you think?